# Request
# (empty)
---
# Response
string[] joint_names
float64[] angles